app/script/localization/webapp.js

```diff
@@ -614,6 +614,9 @@ z.string.preferencesAVMicrophone = 'Microphone';
 z.string.preferencesAVPermissionDetail = 'Enable from your browser Preferences';
 z.string.preferencesAVSpeakers = 'Speakers';
 z.string.preferencesAVTemporaryDisclaimer = 'Guests can’t start video conferences. Select the camera to use if you join one.';
+z.string.preferencesAVNoCamera = 'Wire doesn’t have access to the camera.<br><a href="https://support.wire.com/hc/articles/202935412" data-uie-name="go-no-camera-faq" target="_blank" rel="noopener noreferrer">Read this support article</a> to find out how to fix it.';
```

Comment:
It doesn't seem right to have html markup in the translations (with knowledge of the `uie-name`). I believe we have something to replace fake markups to html in the template, isn't it?